MSAL/test/unit/native_auth/public/MSALNativeAuthPublicClientApplicationTest.swift

```diff
@@ -95,6 +95,25 @@ final class MSALNativeAuthPublicClientApplicationTest: XCTestCase {
         XCTAssertEqual(delegate.channelTargetType, .email)
         XCTAssertEqual(delegate.codeLength, 1)
     }
+    
+    func testSignUpPassword_whenNoAttributesAreSpecified_AttributesShouldBeNil() {
+        let exp = expectation(description: "sign-up public interface")
+        let delegate = SignUpPasswordStartDelegateSpy(expectation: exp)
+
+        let expectedResult: SignUpPasswordStartResult = .codeRequired(
+            newState: .init(controller: controllerFactoryMock.signUpController, username: "", flowToken: "flowToken"),
+            sentTo: "sentTo",
+            channelTargetType: .email,
+            codeLength: 1
+        )
+        controllerFactoryMock.signUpController.startPasswordResult = .init(expectedResult)
+
+        sut.signUpUsingPassword(username: "correct", password: "correct", delegate: delegate)
+
+        wait(for: [exp])
+
```

Comment:
Maybe add a `XCTAssertNotNil(controllerFactoryMock.signUpController.signUpStartRequestParameters)` check so we make sure the it's not the `signUpStartRequestParameters` being nil (as in forgot to be set in the test which would be a false positive) ?
Same for below

Reply:
Yeah, make sense. Done ;)